x-pack/plugins/observability_log_explorer/public/state_machines/origin_interpreter/src/defaults.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const DEFAULT_CONTEXT = undefined;
```

Comment:
The machine is setup in a way that it could easily support future context so that we have parity with our other state machines.